src/sentry/features/temporary.py

```diff
@@ -44,6 +44,8 @@ def register_temporary_features(manager: FeatureManager):
 
     # Enables activated alert rules
     manager.add("organizations:activated-alert-rules", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
+    # Enable AI Issue Summary feture on the Issue Details page.
+    manager.add("organizations:ai-summary", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
```

Comment:
nit: probably want to add `issue` prefix to make it more descriptive

Reply:
I omitted `issue` because I think it would be simpler to just use the same flag for all summaries together, such as session replay summaries etc as well.

Reply:
But would you be rolling out issue and replay summaries at the same time? If so this is good but if we want to roll them out at different times, would be better to have multiple

Reply:
Hmm, given that any sub component summary like replay summaries will feed into the main issue summary, this one is like the main flag. I don't see a world where we would have sub component summaries but not the main issue summary for example, but I can see one where we have issue summary but not component summaries.

What do you think about the approach where this one controls all summaries in general and in the case that we do want to roll out separately, we have a separate `organizations:ai-replay-summary`

Reply:
That seems fine to me